modules/backup-and-disaster-recovery.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * service_description/rhacs-cloud-service-service-description.adoc
+:_content-type: CONCEPT
+[id="backup-disaster-recovery_{context}"]
+= Backup and disaster recovery
+
+The {product-title-managed-short} Disaster Recovery strategy includes backups of database and any customization. This also applies to customer data stored in the Central database. Recovery time will vary based on the number of appliances and database sizes; however, because the appliances can be clustered and distributed, the RTO can be reduced upfront with proper architecture planning.
```

Comment:
```suggestion
The {product-title-managed-short} Disaster Recovery strategy includes backups of database and any customization. This also applies to customer data stored in the Central database. Recovery time can vary based on the number of appliances and database sizes; however, because the appliances can be clustered and distributed, the RTO can be reduced upfront with proper architecture planning.
```